Lib/pdb.py

```diff
@@ -96,7 +97,13 @@ class Restart(Exception):
     pass
 
 __all__ = ["run", "pm", "Pdb", "runeval", "runctx", "runcall", "set_trace",
-           "post_mortem", "help"]
+           "post_mortem", "help", "PdbInvokeType"]
+
+
+class PdbInvokeType(enum.Enum):
+    """Enum to specify the type of invocation for the debugger."""
+    CommandLine = 1
+    InlineBreakpoint = 2
```

Comment:
Actually since you're using a proper enum maybe an 'unknown' option, as you suggested, makes more sense.

Reply:
Yeah I thought so too. I also used string instead a meaningless number as the value. Do you think I can work on docs and tests now?

Reply:
> Do you think I can work on docs and tests now?

Yes.